test/liquid_handle_dispense_test.py

```diff
@@ -856,6 +861,170 @@ def test_tempest_bad_chip_param(self):
                 nozzle="abc",
             )
 
+    def test_mantis_default_params(self):
+        """Tests mantis default params"""
+        instruction = self.protocol.liquid_handle_dispense(
+            source=self.tube.well(0),
+            destination=self.flat.well(0),
+            volume="5:uL",
+            liquid=ProteinBuffer,
+        )
+
+        assert "mode_params" not in instruction.data
+
+        instruction = self.protocol.liquid_handle_dispense(
+            source=self.tube.well(0),
+            destination=self.flat.well(0),
+            volume="5:uL",
+            liquid=ProteinBuffer,
+            device="x_mantis",
```

Comment:
it might seem a bit verbose, but i think we should include rows and columns explicitly here, instead of relying on the method to set it

Reply:
ok!